src/ui/utils/helpers.ts

```diff
@@ -56,3 +58,7 @@ export const matchDatasetFieldFilter = (
             (dlDebugMode && guid?.includes(filterValue)),
     );
 };
+
+export const getDatasetLabelValue = (key: string) => {
```

Comment:
Why this helper is located out of dataset unit?

Reply:
Because it used in https://github.com/datalens-tech/datalens-ui/blob/befddc7911dd1c6231f55402a3489006e9698954/src/ui/utils/getTypeSelectOptions.tsx#L12